app/src/test/java/com/distillery/interview/util/LiveDataTestUtil.kt

```diff
@@ -0,0 +1,39 @@
+package com.distillery.interview.util
+
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+import java.util.concurrent.TimeUnit
+
+/**
+ * Gets the value of a [LiveData] or waits for it to have one, with a timeout.
+ *
+ * Use this extension from host-side (JVM) tests. It's recommended to use it alongside
+ * `InstantTaskExecutorRule` or a similar mechanism to execute tasks synchronously.
+ */
+fun <T> LiveData<T>.getOrAwaitValue(
```

Comment:
Can we check if observeForTesting extension is needed and why we need it or why not?